.gitignore

```diff
@@ -5,7 +5,6 @@ repository
 .vagrant
 keyrings
 /tmp
-.idea
```

Comment:
Editor files should be kept out of project gitignores. See https://stratus3d.com/blog/2018/06/03/stop-excluding-editor-temp-files-in-gitignore/